packages/program-boilerplate/src/types/rpc.ts

```diff
@@ -45,6 +45,7 @@ export type ProgramTriggerBody = {
   activeTrigger: any;
   program: any;
   tenant: {
+    impactBrandId: number | undefined | null;
```

Comment:
I think this should be a string instead of a number. The backend uses graphql to get this value and graphql returns a string version of the brand id. Although the program only uses this value to check if it exists and determine flavor its probably worth being accurate in the types.